concordium-consensus/src/Concordium/KonsensusV1/TreeState/Implementation.hs

```diff
@@ -301,9 +305,13 @@ mkInitialSkovData ::
     Duration ->
     -- |Bakers at the genesis block
     EpochBakers ->
+    -- |'TransactionTable' to initialize the 'SkovData' with.
+    TransactionTable ->
+    -- |'PendingTransactionTable' to initialize the 'SkovData' with.
```

Comment:
Please document here like we do for the V0 consensus, what this pending table must satisfy. This is quite tricky and easy to miss.